changelog/7808.improvement.rst

```diff
@@ -0,0 +1 @@
+pytest now supports python3.6+ only.
```

Comment:
Should we consider this a breaking change perhaps?

Reply:
I was on the fence, since we're going to release this in a minor version 🤔

Reply:
Yeah, but I think there's precedence: we removed a bunch of features in 6.1, and also [5.4](https://docs.pytest.org/en/stable/changelog.html#pytest-5-4-0-2020-03-12).

Reply:
done!